web/themes/custom/pineapple/components/_card.twig

```diff
@@ -24,7 +24,13 @@
         {% endfor %}
       </div>
     {% endif %}
-    <h2><a href="{{ url }}" rel="bookmark">{{ title }}</a></h2>
+    {% set title_link %}
+      <a href="{{ url }}" rel="bookmark">{{ title }}</a>
+    {% endset %}
+    {% include "@components/_heading.twig" with {
+      heading_level: '2',
```

Comment:
Really minor but I'm sure you can have integers without single or double quotes to convert them into  strings.